### PR TITLE
Pr/encode decode json

### PR DESCRIPTION
**Pull request by episodeiv**
As @xsawyerx noted in issue #1262, encode_json/decode_json should be available and recommended. This is my attempt at implementing this.

### CODE REVIEW

File: lib/Dancer2/Cookbook.pod [hunk 1]
```diff
@@ -120,7 +120,7 @@ to generate a graph and expects to get its data, which is in the form
 of wordcount from an AJAX call.
 
 For the graph, we need the url I</data> to return a JSON representation
-of the wordcount data. Dancer infact has a C<to_json()> function that takes
+of the wordcount data. Dancer infact has a C<encode_json()> function that takes
```
Comment (xsawyerx) [inline lib/Dancer2/Cookbook.pod]: Whoa.
Extra typo found: `infact` -> `in fact`. Extra points if you also fix that, please. :)
Also, "wordcount" should probably be "word count", no?

File: lib/Dancer2/Manual.pod [hunk 2]
```diff
@@ -2422,11 +2432,11 @@ the method of your request. Use with caution.
 
 Deserializes a Data::Dumper structure.
 
-=head2 from_json ($structure, \%options)
+=head2 from_json ($string, \%options)
```
Comment (xsawyerx) [inline lib/Dancer2/Manual.pod]: Nice catch!

File: lib/Dancer2/Manual.pod [hunk 3]
```diff
@@ -2422,11 +2432,11 @@ the method of your request. Use with caution.
...
 
-Deserializes a JSON structure. Can receive optional arguments. Those
-arguments are valid L<JSON> arguments to change the behaviour of the default
-C<JSON::from_json> function.
+Deserializes a JSON structure from a string. You should probably use
```
Comment (xsawyerx) [inline lib/Dancer2/Manual.pod]: :+1: I'm happy you added this recommendation here.

**Comment by xsawyerx** [comment]
This is a :+1: from me. I would just like the two typos caught fixed, if possible. We could also do it after but we're already here. :)

**Comment by episodeiv** [comment]
Sure thing! Didn't want to edit around too much to keep this small.

**Comment by racke** [comment]
:+1:

**Comment by SysPete** [comment]
:+1:

**Comment by xsawyerx** [comment]
Merged,  thanks!

**Pull request merged**